graphene_django/filter/tests/test_fields.py

```diff
@@ -671,7 +671,7 @@ def resolve_all_reporters(self, info, **args):
     schema = Schema(query=Query)
     query = """
         query NodeFilteringQuery {
-            allReporters(limit: 1) {
+            allReporters(limit: "1") {
```

Comment:
@zbyte64 I couldn't understand why this change is necessary and i can use some help :slightly_smiling_face: 

The filter here uses `NumberFilter` from django-filters and it expects a decimal input. `Decimal(1) == Decimal("1")` is true but when that input is numeric 1, the test fails with 

```
FAILED graphene_django/filter/tests/test_fields.py::test_should_query_filter_node_limit - assert not [GraphQLError('Argument "limit" has invalid value 1.\nExpected type "Decimal", found 1.')]
```

Reply:
I experimented with adding the following above line 671: 

```
assert Query.all_reporters.filtering_args["limit"].type is Decimal, str(
        Query.all_reporters.filtering_args["limit"].type
    )
 ```
 
 Interestingly this line did not fail. So we know we're generating the right graphene type at least.

Reply:
It is easier to get a stack trace with v3 than v2, still working on this :/

Reply:
Something interesting happened when I branched and rebased off v3, that particular error went away but then it stopped applying the limit, `filter_limit` no longer gets called. BUT if I then switch the form conversion back to float the method gets called by django filter. Digging further the args being passed into `DjangoFilterConnectoinField.resolve_queryset` as `limit` as `None` when we convert to Decimal but not Float.

Reply:
Traced the issue to graphene, I'll be sending them a PR in a moment

Reply:
This should fix the issue we were seeing: https://github.com/graphql-python/graphene/pull/1295